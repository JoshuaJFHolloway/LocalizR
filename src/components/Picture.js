import React from 'react';

const Picture = props => {

    return (
      <div>
        <button onClick={props.handlePictureClicked}><img src="https://cdn2.iconfinder.com/data/icons/Flag/134/Spain.png" alt="themeimage" width="150"/> </button>
      </div>
    );
};

export default Picture;
