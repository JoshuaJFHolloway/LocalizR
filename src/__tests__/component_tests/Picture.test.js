import React from 'react';
import { shallow } from 'enzyme';
import Picture from '../../components/Picture';

describe('Picture', () => {
    let picture = shallow(<Picture />);

    it("renders correctly",() => {
        expect(picture).toMatchSnapshot();
    });

});


