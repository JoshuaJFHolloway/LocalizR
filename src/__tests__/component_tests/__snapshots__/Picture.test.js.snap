// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Picture renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Picture />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <button>
        <img
                alt="themeimage"
                onClick={undefined}
                src="https://cdn2.iconfinder.com/data/icons/Flag/134/Spain.png"
                width="150"
        />
         
</button>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="themeimage"
            onClick={undefined}
            src="https://cdn2.iconfinder.com/data/icons/Flag/134/Spain.png"
            width="150"
/>,
          " ",
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "themeimage",
            "onClick": undefined,
            "src": "https://cdn2.iconfinder.com/data/icons/Flag/134/Spain.png",
            "width": "150",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        " ",
      ],
      "type": "button",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <button>
          <img
                    alt="themeimage"
                    onClick={undefined}
                    src="https://cdn2.iconfinder.com/data/icons/Flag/134/Spain.png"
                    width="150"
          />
           
</button>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="themeimage"
              onClick={undefined}
              src="https://cdn2.iconfinder.com/data/icons/Flag/134/Spain.png"
              width="150"
/>,
            " ",
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "themeimage",
              "onClick": undefined,
              "src": "https://cdn2.iconfinder.com/data/icons/Flag/134/Spain.png",
              "width": "150",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          " ",
        ],
        "type": "button",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
